tests/test_qr.py

```diff
@@ -0,0 +1,25 @@
+import jax
+import jax.numpy as jnp
+import numpy as np
+from numpy.testing import assert_array_almost_equal as aaae
+from skillmodels.qr import qr_gpu
+
+def test_qr():
+    factorized = np.random.uniform(low=-1, high=3, size=(7, 7))
```

Comment:
Please fix the seed (using modern numpy) -- just in case we ever get a failure, we want to be able to reproduce it.